debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -81,6 +82,16 @@
             .withValidation(Field::isRequired)
             .withDescription("Name of the XStream Out server to connect to.");
 
+    public static final Field SNAPSHOT_MODE = Field.create("snapshot.mode")
```

Comment:
This should belong to the parent class

Reply:
I was wondering that, too, but then I figured that the actual snapshotting modes are different between the different connectors. Hence the idea of `SnapshottingTask`, based on its specific snapshotting modes and the previous offset (if any), a specific sub-class returns this descriptor which indicates whether a snapshot of schema, data or both is needed.